website/www/site/content/en/contribute/release-guide.md

```diff
@@ -567,6 +572,12 @@ See the source of the script for more details, or to run commands manually in ca
      * Copy python doc into beam-site
      * Copy java doc into beam-site
 
+
+* **Mitigate GPG Out Of Memory Error**
+  * gpg-agent might through Out Of Memory error. You can kill the gpg-agent to mitigate this issue.
```

Comment:
```suggestion
  * gpg-agent might throw Out Of Memory error. You can kill the gpg-agent to mitigate this issue.
```

Reply:
Is there a jira for this? killing the agent is a workaround, but we should also plan to fix this permanently.

Reply:
I think I saw this as well, but I was also having issues with the gpg passphrase prompt appearing while gradle was running (in ssh, garbling the output of both gradle and gpg and the passphrase prompt was not reading all the characters).

My solution was to use `gradlew  --no-parallel` which also solved the OOM.

Reply:
https://github.com/apache/beam/pull/15544/files#diff-b47163558f49ed59a8a91deaa893c4bae776bfc9bd6249a64814ed7acfcc2621R181